### ISSUE TITLE
Sleeping in a bed resets /back history 

### ISSUE THREAD

**Issue by inboxinstinct** [0PandaDEV/Nextron#99]
Player warps home to sleep. 
Player sleeps.
Player attempts to return with /back.
Player is teleported to the bed.

**Comment by 0PandaDEV** [comment]
Yeah, the back command is a bit buggy currently ill take a look into that

Would you be up to give me your configuration folder for testing purposes because I'm rewriting the config system to use a SQLite database instead of YAML files. The folder i need is the one in your plugins folder of your server called Nextron.

**Comment by inboxinstinct** [comment]
[Nextron.zip](https://github.com/user-attachments/files/16741497/Nextron.zip)

Here ya go

**Comment by 0PandaDEV** [comment]
https://github.com/0PandaDEV/Nextron/commit/bfffd5e713f81b8d8dc5a97779b268be56c1e7fd thanks for the testing data
